solidity/random-beacon/contracts/RandomBeaconGovernance.sol

```diff
@@ -58,6 +58,12 @@ contract RandomBeaconGovernance is Ownable {
     uint256 public newMaliciousDkgResultSlashingAmount;
     uint256 public maliciousDkgResultSlashingAmountChangeInitiated;
 
+    uint256 public newSortitionPoolRewardsBanDuration;
+    uint256 public sortitionPoolRewardsBanDurationChangeInitiated;
+
+    uint256 public newRelayEntryTimeoutNotificationRewardMultiplier;
+    uint256 public relayEntryTimeoutNotificationRewardMultiplierChangeInitiated;
```

Comment:
We should list these two parameters in the comment above `STANDARD_PARAMETER_GOVERNANCE_DELAY`.

Reply:
See https://github.com/keep-network/keep-core/pull/2707/commits/6be521c1ffbd85c473e05aee899ab786382f4eba